frontier/views/home.py

```diff
@@ -9,12 +9,16 @@
 from hotel.models.quotedprice import QuotedPrice
 from hotel.models.room import Room
 
-logger = logging.getLogger('trazip')
+logger_test = logging.getLogger("trazip.frontier.test")
 
 def home(request):
     try:
-        city_1 = City.objects.get(pk=1)
-        logger.debug('Getting city object {}'.format(repr(city_1)))
+        city_1 = City.objects.get(ctrip_id=1)
+        print("cccccccccccccccccccc")
```

Comment:
Please put some meaningful log message here even though it's more like a experiment.